### DISCUSSION TITLE
Question on bioportal metric classes with no definition

### DISCUSSION THREAD

**Discussion by drocha3004** [geneontology/helpdesk#509] [General]
I had a question about the bioportal statistics metric on 

51,567 classes and 3,646 classes with no definition (According to bioportal). Do you know what would be the reason for bioportal to put that metric? Is this correct?

https://bioportal.bioontology.org/ontologies/GO

![Screenshot 2024-11-18 151616](https://github.com/user-attachments/assets/806010ca-ac27-4c60-b911-062abda4f6fc)

**Comment by balhoff** [comment]
It looks correct to me. These are all obsolete classes.

**Comment by drocha3004** [comment]
ok thanks so much
